lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -121,12 +141,23 @@ _InsertionLocation _insertionLocationForPackageImport(
     // No directive to insert relative to; insert before the first member or
     // at the beginning of the file.
     return _InsertionLocation(unit.declarations.firstOrNull?.offset ?? 0,
-        trailingNewlineCount: 2);
+        trailingNewlineCount: 2, usePackageImports: hasPackageImports);
+  }
+
+  for (final importDirective in imports) {
+    final uriContent = importDirective.uri.stringValue;
+    if (uriContent != null) {
+      final uri = Uri.parse(uriContent);
```

Comment:
I think it's also possible for the scheme to be 'dart:'. I handled that in the comment above.

Reply:
The above code has handled all the cases, for specific `uri.scheme != 'package'` I have done it separately and `hasPackageImports=false `so It working here and above condition as well.  also the `bool usePackageImports` are getting updated in return.